test/spec/services/membership/membershipSpec.js

```diff
@@ -87,27 +87,14 @@ describe('MembershipService', function() {
     it('should filter out system-only roles', function() {
       var fakeList = [
         // specific roles filter will explicitly pass
-        {metadata: {name: 'system:image-puller'}},
-        {metadata: {name: 'system:image-pusher'}},
-        {metadata: {name: 'system:image-builder'}},
-        {metadata: {name: 'system:deployer'}},
-        // roles the filter will explicitly reject
-        {metadata: {name: 'cluster-magical-scary-role'}},
-        {metadata: {name: 'system:special-scary-role'}},
-        {metadata: {name: 'registry-ninja'}},
-        {metadata: {name: 'self-destruction'}},
-        // ignored
-        {metadata: {name: 'admin'}},
-        {metadata: {name: 'edit'}}
+        {metadata: {name: 'system-only-role', annotations: {'authorization.openshift.io/system-only': 'true'}}},
```

Comment:
Yeah ill add a few more examples of things that wont actually pass the filter, just to be clear.